internal/sync/cache.go

```diff
@@ -120,7 +122,36 @@ func NewLiveStateCache(
 	}
 }
 
+func (c *liveStateCache) invalidate(cacheSettings cacheSettings) {
+	c.logger.Info("invalidating live state cache")
+	c.lock.Lock()
+	c.cacheSettings = cacheSettings
+	cluster := c.cluster
```

Comment:
is this variable not used?

Reply:
you are asking about `cluster`? it is used at L132 .

Reply:
oh, I see - the way the indentation is showing up in my browser on line 132 is weird because it's trying to squeeze the whole line in, and it looks like the start of a new function.